### PR TITLE
Revert removal of gtk_alignment

### PR DESCRIPTION
**Pull request by oz123**
Fix for the issue in #152

### CODE REVIEW

File: mate-screenshot/src/mate-screenshot.c [hunk 1]
```diff
@@ -315,7 +315,9 @@ create_effects_frame (GtkWidget   *outer_vbox,
                       const gchar *frame_title)
 {
   GtkWidget *main_vbox, *vbox, *hbox;
+#if GTK_CHECK_VERSION (3, 0, 0)
```
Comment (monsta) [inline mate-screenshot/src/mate-screenshot.c]: @oz123: should be ```#if !GTK_CHECK_VERSION``` probably?

Comment (oz123) [inline mate-screenshot/src/mate-screenshot.c]: Oh. Sorry. Yes you are right

**Comment by monsta** [comment]
You can actually squash commits in your own branch - it should be fine, it won't close the PR or anything :smile:

**Comment by monsta** [comment]
Ok, looks better in GTK+3 now, but somehow the image isn't top-aligned like in GTK+2...

almost-there

**Comment by oz123** [comment]
@monsta , just for the record, I am uploading here a screenshot of both versions:
gtk3-vs-gtk2

To the right is the GTK3 version, where the image is centered. IMHO it looks better like this and it's not a breakage of something... just a small improvement (we can argue about that, since it's a matter of taste). If you think no changes at all should be introduced, I can change the image to
stick to the top.

**Comment by monsta** [comment]
Yes, I'd prefer to not introduce UI changes when we're just fixing deprecations... might be a surprise for some users. We can always discuss this change later.

**Comment by raveit65** [comment]
Btw. here is an example to replace GtkAlignment for Gtk3.
https://github.com/mate-desktop/mate-notification-daemon/commit/75965a8aa6c329974dec8d7a7d666c95a08d5b1f

**Comment by raveit65** [comment]
+1 for using the GTK3 version, where the image is centered.
This is a fixing PR for previous UI changes PR, so this is valid if all agree here.
We should not always ask the whole world for a little UI change.
We are Mate.......
And Mate 1.16 with Gtk3 will differ in details compare to gtk2 version.

**Comment by oz123** [comment]
@raveit65 , @monsta, now the image is shown at the top like in the GTK+2 UI. I personally find it better with a center image, but let's merge this like this and we can change the cosmetics in 1.18.

**Comment by monsta** [comment]
Thanks, squashed and merged in https://github.com/mate-desktop/mate-utils/commit/7d1301360c116adbcf5584636d515434ceee6607.

**Pull request closed without merging**